python/ray/tests/test_gcs_fault_tolerance.py

```diff
@@ -126,6 +127,28 @@ def condition():
     wait_for_condition(condition, timeout=10)
 
 
+@pytest.mark.parametrize(
+    "ray_start_regular", [
+        generate_system_config_map(
+            num_heartbeats_timeout=20, ping_gcs_rpc_server_max_retries=60)
+    ],
+    indirect=True)
+def test_del_actor_after_gcs_server_restart(ray_start_regular):
+    actor = Increase.options(name="abc").remote()
+    result = ray.get(actor.method.remote(1))
+    assert result == 3
+
+    ray.worker._global_node.kill_gcs_server()
+    ray.worker._global_node.start_gcs_server()
+
+    del actor
+    time.sleep(1)
```

Comment:
Can you call wait_for_condition on ray.actors table instead?

Reply:
Good idea, thanks